ContentValidation/ExtraLabelValidation.cs

```diff
@@ -41,12 +44,17 @@ public async Task<TResult> Validate(string testLink)
             "&apos"
         };
 
+        //Create browser instance.
         var browser = await _playwright.Chromium.LaunchAsync(new BrowserTypeLaunchOptions { Headless = true });
         var page = await browser.NewPageAsync();
         await page.GotoAsync(testLink);
 
         var text = await page.Locator("html").InnerTextAsync();
 
+
+        // Iterate through labelList and check if the page content contains any of the tags. If any tags are found, add them to the errorList.
+        int sum = 0;
+        string ErrorInfo = "Extra label found: ";
```

Comment:
Update to: errorInfo.

Reply:
@zedy-wj , I have updated by your comments, please re-review.